map2loop/mapdata.py

```diff
@@ -723,7 +732,10 @@ def parse_structure_map(self) -> tuple:
                 structure["DIPDIR"] = self.raw_data[Datatype.STRUCTURE][config["dipdir_column"]]
         else:
             print(f"Structure map does not contain dipdir_column '{config['dipdir_column']}'")
-
+                    
+        # Ensure all DIPDIR values are within [0, 360]
+        structure["DIPDIR"] = structure["DIPDIR"] % 360.0
```

Comment:
What if its negative?

Reply:
It would still return the modulus of 360.